_layouts/landing.html

```diff
@@ -118,25 +118,42 @@ <h5 class="title">Reference</h5>
+              <li><a data-toggle="tab" data-target="#features">Get Support</a></li>
+              <li><a data-toggle="tab" data-target="#faqs">Browse FAQs</a></li>
+              </ul>
+              <div class="tab-content">
+              <div id="includes" class="tab-pane fade in active" markdown="1">
+               <p><a href="/engine/reference/builder/">Write a Dockerfile</a></p>
+               <p><a href="/network/">Manage container networking</a></p>
+               <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/04_sharing_app/">Share my image on Docker Hub</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/daemon/">Configure the Docker daemon</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
+              </div>
+              <div id="features" class="tab-pane fade" markdown="1">
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/support/">Get support</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://dockercommunity.slack.com/">Docker community Slack</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://www.docker.com/company/contact/">Contact Docker</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://github.com/docker/for-win">test</a></div>
+              <div class="col-xs-12 col-md-6"><a href="Docker Desktop for Mac GitHub repo">test</a></div></div>
+              <div id="faqs" class="tab-pane fade" markdown="1">
+              <div class="col-xs-12 col-md-6"><a href="https://github.com/docker/for-linux">test</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/install/windows-install/#system-requirements">test</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/faqs/general">Docker Desktop</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/faqs/macfaqs/">Docker Desktop for Mac</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/faqs/windowsfaqs/">Docker Desktop for Windows</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/faqs/linuxfaqs/">Docker Desktop for Linux</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/single-sign-on/faqs/">Docker Hub Single Sign-on</a></div>
```

Comment:
Should this be Docker Single Sign-on as SSO isn't just about Docker Hub?

Reply:
Updated to Docker Single Sign-on